ts/card-info/CardInfo.svelte

```diff
@@ -3,108 +3,53 @@ Copyright: Ankitects Pty Ltd and contributors
 License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 -->
 <script lang="ts">
-    import * as tr2 from "../lib/ftl";
-    import { Stats, unwrapOptionalNumber } from "../lib/proto";
-    import { Timestamp, timeSpan, DAY } from "../lib/time";
+    import * as tr from "../lib/ftl";
+    import type { Stats } from "../lib/proto";
+    import { getCardStats } from "./lib";
+    import CardStats from "./CardStats.svelte";
     import Revlog from "./Revlog.svelte";
 
-    export let stats: Stats.CardStatsResponse;
+    export let cardId: number | undefined = undefined;
+    export let includeRevlog: boolean | undefined = undefined;
 
-    function dateString(timestamp: number): string {
-        return new Timestamp(timestamp).dateString();
-    }
-
-    interface StatsRow {
-        label: string;
-        value: string | number;
-    }
-
-    const statsRows: StatsRow[] = [];
-
-    statsRows.push({ label: tr2.cardStatsAdded(), value: dateString(stats.added) });
-
-    const firstReview = unwrapOptionalNumber(stats.firstReview);
-    if (firstReview !== undefined) {
-        statsRows.push({
-            label: tr2.cardStatsFirstReview(),
-            value: dateString(firstReview),
-        });
-    }
-    const latestReview = unwrapOptionalNumber(stats.latestReview);
-    if (latestReview !== undefined) {
-        statsRows.push({
-            label: tr2.cardStatsLatestReview(),
-            value: dateString(latestReview),
-        });
-    }
-
-    const dueDate = unwrapOptionalNumber(stats.dueDate);
-    if (dueDate !== undefined) {
-        statsRows.push({ label: tr2.statisticsDueDate(), value: dateString(dueDate) });
-    }
-    const duePosition = unwrapOptionalNumber(stats.duePosition);
-    if (duePosition !== undefined) {
-        statsRows.push({
-            label: tr2.cardStatsNewCardPosition(),
-            value: dateString(duePosition),
-        });
-    }
+    let stats: Stats.CardStatsResponse | undefined;
 
-    if (stats.interval) {
-        statsRows.push({
-            label: tr2.cardStatsInterval(),
-            value: timeSpan(stats.interval * DAY),
+    $: if (cardId === undefined) {
+        stats = undefined;
+    } else {
+        const sentCardId = cardId;
```

Comment:
```suggestion
        const requestedCardId = cardId;
```

Bikeshedding, feel free to ignore.